bentoml/_internal/models/model.py

```diff
@@ -18,9 +18,9 @@
 import fs.mirror
 import cloudpickle  # type: ignore (no cloudpickle types)
 from fs.base import FS
-from cattr.gen import override
-from cattr.gen import make_dict_structure_fn
-from cattr.gen import make_dict_unstructure_fn
+from cattr.gen import override  # type: ignore (incomplete cattr types)
+from cattr.gen import make_dict_structure_fn  # type: ignore (incomplete cattr types)
+from cattr.gen import make_dict_unstructure_fn  # type: ignore (incomplete cattr types)
```

Comment:
no need for type ignore here. Since this will be addressed in newer version of cattrs.

Reply:
Sure, but it hasn't been addressed yet.